src/amplitude-client.js

```diff
@@ -79,6 +79,12 @@ AmplitudeClient.prototype.init = function init(apiKey, opt_userId, opt_config, o
     this.options.apiKey = apiKey;
     this._storageSuffix = '_' + apiKey + this._legacyStorageSuffix;
 
+    var hasExistingCookie = !!this.cookieStorage.get(this.options.cookieName + this._storageSuffix);
+    if (opt_config && opt_config.deferInitialization && !hasExistingCookie) {
```

Comment:
note for existing users, they won't get the option to defer tracking because they will already have a cookie, i guess we just need to communicate that to our customers and the people that requested this feature

Reply:
yes, this will only work on new users without existing cookies and they have to add the config flag